static/css/index.css

```
@@ -2,259 +2,259 @@
+    padding: 0;
+    /*background: #fff;*/
+    background: transparent;
+    color: #222;
+    font-family: 'Segoe UI', Arial, sans-serif;
+    pointer-events: none;
+    /* 允许鼠标事件穿透 */
+    overflow: hidden;
+    /* 防止出现滚动条 */
 }
 
 @keyframes fadeIn {
```

Stylelint (17.3.0)

[error] Expected keyframe name "fadeIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)